### PR TITLE
sveltekit

### PR DESCRIPTION
**Pull request by rishi-raj-jain**

### CODE REVIEW

File: blog/2024-04-29-secure-sveltekit.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+const realm = "shared-deployment-001";
+
+const kcConfig = {
+  issuer: `https://usw2.auth.ac/auth/realms/${realm}`,
+  clientId: "reg-example-1",
+  clientSecret: "CLIENT_SECRET", // Paste "Client secret" here. Use Environment variables in prod
+};
+```
+
+Those are used to popluate the config for the provider `Keycloak` in Auth.js:
```
Comment (pnzrr) [inline blog/2024-04-29-secure-sveltekit.mdx:72]: `auth.ts`

**Review by pnzrr — CHANGES_REQUESTED** [review]
Please add more instructions explaining the app a bit more. See the Nuxt and Vue for examples of talking about the pages/components more

https://phasetwo.io/blog/instant-user-managemenet-and-sso-for-vue/
https://phasetwo.io/blog/instant-user-managemenet-and-sso-for-nuxt/

File: blog/2024-04-29-secure-sveltekit.mdx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+```tsx
+import type { LayoutServerLoad } from "./$types.js";
+
+export const load: LayoutServerLoad = async (event) => {
+  const session = await event.locals.getSession();
+  return {
+    session,
+  };
+};
+```
```
Comment (pnzrr) [inline blog/2024-04-29-secure-sveltekit.mdx:106]: @rishi-raj-jain let's add the next few steps following the data through the application: 

Once layout sets that value, +page uses it 

```
<User
        data={{
          user: data?.session?.user,
          status: Boolean(data?.session),
        }}
      />
```

The user component can then determine whether to show Sign In or Out based on the value of the user object.

**Review by pnzrr — APPROVED** [review]

File: blog/2024-04-29-secure-sveltekit.mdx (hunk 2 above)
Comment (rishi-raj-jain) [reply blog/2024-04-29-secure-sveltekit.mdx:106]: got it adding

**Review by pnzrr — CHANGES_REQUESTED** [review]
Meant to request change

**Review by pnzrr — APPROVED** [review]

**Pull request merged**